stixcore/data/idb/idbVersionHistory.json

```
@@ -30,9 +30,16 @@
         {
                 "version":"2.26.35",
                 "aswVersion":181,
+                "validityPeriodUTC":["2021-06-23T14:38:41.609", "2021-012-010T00:00:00.000"],
+                "validityPeriodOBT":[{"coarse":677774250, "fine":0}, {"coarse":692409503, "fine":537384}]
+
+        },
+        {
+                "version":"2.26.36",
+                "aswVersion":183,
                 "comment": "return this for the next future",
-                "validityPeriodUTC":["2021-06-23T14:38:41.609", "2136-02-07T08:11:44.778"],
-                "validityPeriodOBT":[{"coarse":677774250, "fine":0}, {"coarse":4294967295, "fine":0}]
+                "validityPeriodUTC":["2021-012-010T00:00:00.000", "2136-02-07T08:11:44.778"],
```

I don't now if this is causing the problem
```suggestion
                "validityPeriodUTC":["2021-12-10T00:00:00.000", "2136-02-07T08:11:44.778"],
```